src/content/transformers/onboardingPromptsTransformer.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  Transformer,
+  OnboardingPromptsAugmented,
+  OnboardingPromptsRaw
+} from '../content.types';
+
+export const onboardingPromptsTransformer: Transformer<
+  OnboardingPromptsRaw,
+  OnboardingPromptsAugmented
+> = (prompts: OnboardingPromptsRaw[]): OnboardingPromptsAugmented[] => {
+  return prompts.map((prompt) => ({
+    ...prompt.data
+  }))
+}
```

Comment:
👏 for use of `.map()` here!

FYI: no need for `return` here w/ the use of an arrow function..